src/Elastic.Apm/Api/Context.cs

```diff
@@ -1,45 +1,50 @@
-using System;

```

Comment:
Sorry about the lots of changes here - this is again our line ending issue. The only real change here is that I added `public Dictionary<string, string> Custom => _custom.Value;` and `public bool ShouldSerializeCustom() => _custom.IsValueCreated && Custom.Count > 0;`